README.md

```diff
@@ -162,6 +162,7 @@ The Almost Netflix series is a tutorial for building a Netflix clone with Appwri
 * [How To Generate and Store Map Previews in Ruby with Appwrite](https://dev.to/oliverpham/how-to-generate-and-store-map-previews-in-ruby-with-appwrite-43ha)
 * [What is Appwrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)
 * [How Appwrite real-time can make your CMS changes instant](https://sakshiuppoor.hashnode.dev/how-appwrite-real-time-can-make-your-cms-changes-instant)
+* [Discord crypto price alert bot powered by Appwrite](https://github.com/RizkyRajitha/appwritediscorddemo)
```

Comment:
@RizkyRajitha Great PR! 🤯 We left some comments during the review, please check them out. 

1. Let's put this under `### Appwrite Services` instead since this is a showcase of Appwrite Functions.
2. Does your function logic need to handle promises with `then`? Wouldn't `await` be easier?
3. It would be better to not call `process.exit()` so that the function stays active and doesn't require a cold start during the next execution. It's probably actually better to call `res.json()` on success and throw an exception on error so that the execution shows as completed or failed.